meetpy/meetups/views.py

```diff
@@ -140,3 +143,26 @@ def _send_email_to_admins(self):
 
 class TalkProposalConfirmationView(generic.TemplateView):
     template_name = 'meetups/talkproposal_confirmation.html'
+
+
+class SearchView(generic.TemplateView):
+    template_name='meetups/search_results.html'
+
+    def search(self):
+        query = self.request.GET.get('q')
+        talks = models.Talk.objects.filter(Q(description__icontains=query) | Q(title__contains=query))
+        events = models.Meetup.objects.filter(Q(description__icontains=query))
+        speakers = models.Speaker.objects.filter(Q(biography__icontains=query))
```

Comment:
@olga-black something to consider here. As far I understand by default it is expected that this is using PostgreSQL. I think what would be nice improvement is to use FTS (Full Text Search) capability available in PostgreSQL. Django has the support of that feature here:
https://docs.djangoproject.com/en/2.2/ref/contrib/postgres/search/